### PR TITLE
[Calendar] Adds support for WeekNumbers in Day View

### PR DESCRIPTION
**Pull request by lubber-de**
### Description
- An optional new setting "showWeekNumbers" in day-view adds a column to the left with the appropriate weeknumber 

```javascript
$('#date_calendar')
        .calendar({
            type: 'date',
            showWeekNumbers: true
        })
    ;
```
image

### CODE REVIEW

**Comment by hammy2899** [comment]
That's a good idea and I'm sure a lot of people will use that.

File: src/definitions/modules/calendar.js [hunk 1]
```diff
@@ -19,6 +19,20 @@ window = (typeof window != 'undefined' && window.Math == Math)
     : Function('return this')()
 ;
 
+//support existing momentjs or date-fns library
+Date.prototype.getWeekOfYear = Date.prototype.week || Date.prototype.getWeek || function() {
```
Comment (hammy2899) [inline src/definitions/modules/calendar.js]: Let's move this function into the get methods on line [537](https://github.com/fomantic/Fomantic-UI/blob/3f9852142e61ef1f051ce1411c12d74803dda4e8/src/definitions/modules/calendar.js#L537)

Comment (lubber-de) [reply src/definitions/modules/calendar.js]: Done in latest commit

**Comment by lubber-de** [comment]
@ColinFrick  Fixed week calculation display

**Pull request merged**